lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -133,3 +144,25 @@ def create_symlink_file(self, repo: str, branch: str, location: str = None)  ->
             kwargs["location"] = location
 
         return client.metadata.create_symlink_file(repository=repo, branch=branch, **kwargs)["location"]
+
+    def test_connection(self):
+        "TO do"
+        """Test  Connection"""
+        try:
+            # client = self.get_conn()
+            conn = self.get_connection(self.lakefs_conn_id)
+            import requests
+            import json
+            url = conn.host+"/api/v1/auth/login"
+            payload = json.dumps({
+                "access_key_id": conn.extra_dejson.get("access_key_id", None),
+                "secret_access_key": conn.extra_dejson.get("secret_access_key", None)
+            })
+            headers = {'Content-Type': 'application/json'}
+            response = requests.request("POST", url, headers=headers, data=payload)
+            if response.status_code == 200:
```

Comment:
```suggestion
            if response:
```
There are many ways to succeed in HTTP; requests implements boolean conversion on `response` to allow checking whether it was "okay" in any sense.

Reply:
Alternatively, just
```py
response.raise_for_status()
return True, "Connection successfully tested"
```
here will fail nicely and return a nice error from line 168.

Reply:
ok I will change it.